### PR TITLE
[Purview] Purview GA Azure CLI request

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
---

Onboard Azure CLI for Microsoft.Purview Resource Provider to preform CRUD operations on resources.https://github.com/Azure/azure-cli/issues/19186
closed https://github.com/Azure/azure-cli/issues/19186

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by PARADISSEEKR** [comment]
closed https://github.com/Azure/azure-cli/issues/19186

**Comment by PARADISSEEKR** [comment]
closed [https://github.com/Azure/azure-cli/issues/19186]

**Comment by yonzhan** [comment]
Purview

File: src/purview/azext_purview/manual/custom.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    account_update_parameters = {}
+    if tags is not None:
+        account_update_parameters['tags'] = tags
+    account_update_parameters['properties'] = {}
+    if managed_group_name is not None:
+        account_update_parameters['properties']['managed_resource_group_name'] = managed_group_name
+    if public_network_access is not None:
+        account_update_parameters['properties']['public_network_access'] = public_network_access
+    else:
+        account_update_parameters['properties']['public_network_access'] = "Enabled"
```
Comment (kairu-ms) [inline src/purview/azext_purview/manual/custom.py:80]:
```suggestion
```

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**